lib/decanter/parser/core.rb

```diff
@@ -8,6 +8,10 @@ def self.included(base)
 
       module ClassMethods
 
+        def _parse(name, value, options={})
```

Comment:
I see this called in line 20 - did this move from somewhere else or did it not exist previously?

Reply:
This method didn't exist here previously - it was defined in `ValueParser`. Moving it here ensures that all parsers (including `ArrayParser`, which no longer inherits from `ValueParser`) can still access it. This change also allows `ValueParser` to call `super` and include the singularity check, which doesn’t apply to all parsers.